### PR TITLE
Hyperlink scroll

### PR DESCRIPTION
**Pull request by farisashai**
- fixed scrolling on community page to offset top margin by header height so active section doesn't appear under the header
- fixed weird bug at exactly 900px where mobile navbar would disappear if opened because of overlapping media queries
- simplified scroll to contact (had a weird onClick function that manually scrolled the window because nobody ever gave the footer an id)

still not sure why links from community grid go to wrong location sometimes when we're clicking from a diff page

### CODE REVIEW

**Comment by trulyronak** [comment]
Can you also make it so the mobile navbar goes away after you click on a link

File: src/components/NavigationBar/style.scss [hunk 1]
```diff
@@ -113,9 +113,9 @@
 }
 // Styling for menu based nav bar
 
-@media only screen and (min-width: 900px) {
+@media only screen and (min-width: 900.1px) {
```
Comment (trulyronak) [inline src/components/NavigationBar/style.scss:116]: Why 900.1?

**Review by trulyronak — CHANGES_REQUESTED** [review]
Can you also make it so the mobile navbar goes away after you click on a link

Also if possible can you add all the media query values to a global vars CSS file (just to make it a bit cleaner)

File: src/components/NavigationBar/style.scss (hunk 1 above)
Comment (farisashai) [reply src/components/NavigationBar/style.scss:116]: i was just being really nitpicky. if i set the breakpoint at 900px, it breaks at 900px and if i set it at 901px, it breaks between 900px and 901px (which you can achieve if you zoom in on the page). i wish media queries allowed you to use operators like < vs <= to avoid overlapping selectors but thats not a thing :(

Comment (farisashai) [reply src/components/NavigationBar/style.scss]: nvm ignore that code i found a much easier way to fix it

**Comment by trulyronak** [comment]
Moved the media queries request to #202 since it's a bigger problem that I thought

**Review by trulyronak — APPROVED** [review]
Hype

File: src/components/NavigationBar/index.tsx [hunk 2]
```diff
@@ -13,13 +13,9 @@ const navLinks = [
 
 const NavigationBar: React.FC = () => {
   const [menuState, setMenuState] = useState(false);
-  const toggleMenu = () => {
-    setMenuState(!menuState);
-  };
-  const scrollToContacts = (): void => {
-    window.scrollTo({ top: document.body.scrollHeight });
-    setMenuState(false);
-  };
```
Comment (trulyronak) [inline src/components/NavigationBar/index.tsx:22]: seeking to learn: why did you remove `scrollToContacts`

**Review by trulyronak — APPROVED** [review]
Looks good, a bit curious why we don't need `scrollToContacts`

File: src/components/NavigationBar/index.tsx (hunk 2 above)
Comment (farisashai) [reply src/components/NavigationBar/index.tsx:22]: the 2 things happening here are manually forcing the page to scroll and closing the mobile nav menu. we no longer need to force the page to scroll because the link href works as intended by giving the footer an id. closing the mobile nav menu is handled by toggleMenu because the only time it gets called is when menuState is true
